opencti-platform/opencti-graphql/tests/03-streams/00-Raw/raw-test.js

```diff
@@ -58,8 +58,8 @@ describe('Raw streams tests', () => {
       expect(updateEventsByTypes['marking-definition'].length).toBe(2);
       expect(updateEventsByTypes['campaign'].length).toBe(7);
       expect(updateEventsByTypes['relationship'].length).toBe(8);
-      expect(updateEventsByTypes['identity'].length).toBe(16);
-      expect(updateEventsByTypes['malware'].length).toBe(16);
+      expect(updateEventsByTypes['identity'].length).toBe(18);
```

Comment:
For curiosity, why here we increment by 2 instead of 4 as we have 4 identity in the bundle ? 2 already exist in the platform ?

Reply:
We have 6 updates due to sharing to Oganization:
- 1 Report
- 1 Indicator  + 1 Individual (`Identity`) + 1 System (`Identity`) + 1 Malware + 1 SCO

`City` `Organization` and `Sector` are entities to wich restriction dosen't not apply

Reply:
Ah okok thanks!